ironfish/src/rpc/routes/wallet/exportAccount.ts

```diff
@@ -29,6 +31,7 @@ export const ExportAccountResponseSchema: yup.ObjectSchema<ExportAccountResponse
       .object({
         name: yup.string().defined(),
         spendingKey: yup.string().defined(),
```

Comment:
Isn't this optional now? It might be null

Reply:
It'll be defined, but with value `null`, the way it is currently written.

Reply:
https://github.com/iron-fish/ironfish/blob/38d2528903d3def6be0eaa4eaf400deb541b404c/ironfish/src/rpc/routes/peers/getPeers.ts#L63